components/dashboard/event-card/index.tsx

```diff
@@ -18,6 +18,7 @@ const EventCard: React.FC<Props> = ({ event, project, faded = false }) => {
   return (
     <CardElem>
       <S.Header>
+        {faded && (<S.FadedTag>Proběhlo</S.FadedTag>)}
```

Comment:
Hnida: Myslel jsem původně `faded` jako obecný mechanismus, může být, že někde budem chtít vyšedlou událost z jiných důvodů. Pak by neseděl ten nápis „Proběhlo“. Takže můžem například přejmenovat `faded` na `past`, ale to by bylo trochu divný, jelikož ten příznak nemusí sedět se skutečnými daty. Tak možná zavést samostatnou prop pro ten nápis? Například `note` nebo tak něco, a tím pádem pak `{note && …}`. A můžem kombinovat `note` a `faded` podle potřeby?

Reply:
Jo, to je pravda, šlo by to parametrizovat, vyzkouším!

Reply:
Když o tom dumám, tak `<S.FadedTag text='Proběhlo'></S.FadedTag>` mi nepřijde o moc než `<S.FadedTag>Proběhlo</S.FadedTag>`. Ale tobě jde spíše o ten název `FadedTag`, takže spíše přejmenovat celou tu komponentu např. na `Note`?

Takže celé by to bylo nějak jako:

```react
```

Reply:
Ano, přesně tak!

Reply:
Já jsem to vlastně myslel úplně jinak, takhle asi:

```typescript
interface Props {
   faded?: boolean;
   note?: string;
 }

 const EventCard: React.FC<Props> = ({ past = false, note }) => {
   const CardElem = past ? S.FadedCard : S.Card;
   return (
     <CardElem>
       <S.Header>
         {note && (<S.Note>{note}</S.Note>)}
```

Tedy aby šlo to `faded` a `note` libovolně kombinovat podle potřeb volajícího.